package.json

```diff
@@ -1,17 +1,66 @@
 {
   "name": "roboconf-web-administration",
   "version": "0.0.0",
+  "homepage": "http://roboconf.net",
   "description": "Web administration for Roboconf",
-  "main": "index.js",
+  "keywords": [
+    "cloud",
+    "deployment",
+    "orchestration"
+  ],
   "dependencies": {
     "express": "~3.4.7"
   },
   "devDependencies": {
-    "bower": "~1.3.8"
+    "bower": "~1.3.8",
```

Comment:
You can indeed but on a production server you will download useless dependencies like `gulp` or `karma`. It is good practice in `npm` and `bower` world to separate dependencies. I'm not the product owner so do what you want :wink: